wg-deployment/README.md

```diff
@@ -0,0 +1,39 @@
+<!---
+This is an autogenerated file!
+
+Please do not edit this file directly, but instead make changes to the
+sigs.yaml file in the project root.
+
+To understand how this file is generated, see https://github.com/kubeflow/community/generator/README.md
+--->
+# Deployment Working Group
+
+WG Deployment covers developing, maintaining artifacts to deploy and operate Kubeflow.
+
+The [charter](charter.md) defines the scope and governance of the Deployment Working Group.
+
+## Meetings
+* Regular WG Meeting (Pacific AM): [Tuesdays at 08:00 PT (Pacific Time)]() (biweekly - every other Tuesday). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=08:00&tz=PT%20%28Pacific%20Time%29).
+  * [Meeting notes and Agenda](http://bit.ly/kf-meeting-notes).
+* Regular WG Meeting (Pacific PM): [Wednesdays at 17:30 PT (Pacific Time)]() (biweekly - every other Tuesday). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=17:30&tz=PT%20%28Pacific%20Time%29).
+  * [Meeting notes and Agenda](http://bit.ly/kf-meeting-notes).
+
+## Organizers
```

Comment:
What's the purpose of this section? Is this duplicating the leads listed in the YAML file? Are these folks leads?